playground/src/output/Output.js

```diff
@@ -0,0 +1,99 @@
+import "./Output.css";
+
+import React from "react";
+
+import { Light as SyntaxHighlighter } from "react-syntax-highlighter";
+
+function Tab(props) {
+  return (
+    <button
+      className={`tab-title ${props.parent.tab === props.name ? "active" : ""}`}
+      onClick={() => props.parent.onTabChange(props.name)}
+    >
+      {props.text}
+    </button>
+  );
+}
+
+class Output extends React.Component {
+  state = {
+    justCopied: false,
+  };
+
+  render() {
+    return (
+      <div className="tab">
+        <div className="tab-top">
+          <Tab text="output.sql" name="sql" parent={this.props} />
+          <Tab text="output.arrow" name="arrow" parent={this.props} />
+
+          <div className="spacer"></div>
+
+          <button className="action" onClick={() => this.copyOutput()}>
+            {this.state.justCopied ? "Copied!" : "Copy to clipboard"}
+          </button>
+        </div>
+
+        {this.renderContent()}
+      </div>
+    );
+  }
+
+  renderContent() {
+    if (!this.props.content) {
+      return <div className="tab-content"></div>;
+    }
+    if (this.props.tab === "sql") {
+      return (
+        <SyntaxHighlighter language="sql" useInlineStyles={false}>
+          {this.props.content.sql}
+        </SyntaxHighlighter>
+      );
+    }
+    if (this.props.tab === "arrow" && this.props.content.arrow) {
+      const arrow = this.props.content.arrow;
+
+      const header = arrow.schema.fields.map((f, index) => {
+        return <th key={index}>{f.name}</th>;
+      });
+
+      const data = arrow.toArray().map((x) => [...x]);
+      const rows = data.map((x, index) => {
+        const cells = x.map(([_name, value], index) => (
+          <td key={index}>{"" + value}</td>
```

Comment:
@snth you exactly right. Here I'd have to do a `new Date(value)` when type is timestamp/date.

That could be determined from looking into `arrow.schema.fields[index].type`.

I probably won't get around to doing this, but I'm leaving a comment here for anyone who might want to.